modules/api/src/main/java/org/apache/ignite/lang/IgniteException.java

```diff
@@ -289,4 +288,14 @@ public static IgniteException wrap(Throwable e) {
 
         return new IgniteException(UNKNOWN_ERR, e.getMessage(), e);
     }
+
+    /**
+     * Gets the Ignite error code if the specified throwable is an {@link IgniteException}.
+     *
+     * @param t Throwable.
+     * @return Ignite error code or UNKNOWN_ERR.
+     */
+    public static int getIgniteErrorCode(Throwable t) {
```

Comment:
good change. It will fix [IGNITE-17970](https://issues.apache.org/jira/browse/IGNITE-17970)